config/localization/bundles/src/bundle/org/sakaiproject/localization/bundle/type/types.properties

```diff
@@ -58,6 +58,7 @@ alert.unable		 = Unable to add item
 alert.unable1		 = Unable to add item: {0}
 alert.unknown		 = An error occurred. 
 alert.utf8encoding = Content cannot be encoded in UTF-8.
+alert.zip.filenumber = The ZIP file cannot be expanded because contains more files than the maximum allowed.
```

Comment:
The ZIP file cannot be expanded because it contains more files than the maximum allowed.